ProjectShop/WishList/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+from UserApp.models import User
+
+
+class WishList(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    # products = models.ManyToManyField(Product, related_name='wishlists')
```

Comment:
why this line is commented?

Reply:
Because we haven't merged the product model to our project yet.